crates/extensions/tedge_config_manager/src/lib.rs

```diff
@@ -113,13 +113,17 @@ impl ConfigManagerBuilder {
 
         // create tedge-configuration-plugin.toml
         let example_config = r#"# Add the configurations to be managed
-files = [
-#    { path = '/etc/tedge/tedge.toml' },
-#    { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf', type = 'c8y-bridge.conf' },
-#    { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf', type = 'tedge-mosquitto.conf' },
-#    { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto.conf' },
-#    { path = '/etc/tedge/plugins/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o444 },
-]"#;
+[[files]]
+path = "/etc/tedge/tedge.toml"
```

Comment:
Shouldn't we use the `config_dir` path (e.g. the variable) rather than a hard coded value?